integration-tests/__tests__/expect-async-matcher.test.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * Copyright (c) 2014-present, Facebook, Inc. All rights reserved.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow
+ */
+
+'use strict';
+
+const path = require('path');
+const SkipOnWindows = require('../../scripts/SkipOnWindows');
+const runJest = require('../runJest');
+const {extractSummary} = require('../Utils');
+const dir = path.resolve(__dirname, '../expect-async-matcher');
+
+SkipOnWindows.suite();
```

Comment:
is this needed?

Reply:
@SimenB I used it because of the filesystem differences with the snapshots. I think that the failures example that you provided is doing it for the same reason.